lib/game.rb

```diff
@@ -98,17 +96,21 @@ def place_computer_shot
     elsif @player_board.cells[coordinate].render == "X"
       puts "Their shot on #{coordinate} sunk your ship!"
     end
+    game_over?
+  end
 
-    if @player_cruiser.sunk? && @player_submarine.sunk?
-      @game_over = true
-    end
+  def player_all_sunk?
+    @player_cruiser.sunk? && @player_submarine.sunk?
+  end
+
+  def computer_all_sunk?
+    @computer_cruiser.sunk? && @computer_submarine.sunk?
   end
 
-  def game_over_message
-    if @player_cruiser.sunk? && @player_submarine.sunk?
-      p "HA, You lost! Try again!"
-    else @computer_cruiser.sunk? && @computer_submarine.sunk?
-      p "I want a rematch!"
+  def game_over?
```

Comment:
Nice way to refactor the game over for testing and as well has getting rid of repetitiveness  in the other methods.